hikari/locales.py

```diff
@@ -37,91 +35,91 @@ class Locale(str, enums.Enum):
     """Possible user/guild locales."""
 
     DA = "da"
-    """Language is Danish (da)"""
+    """Danish"""
 
     DE = "de"
-    """Language is German (de)"""
+    """German"""
 
     EN_GB = "en-GB"
-    """Language is English, UK (en-GB)"""
+    """English, UK"""
 
     EN_US = "en-US"
-    """Language is English, US (en-US)"""
+    """English, US"""
 
     ES_ES = "es-ES"
-    """Language is Spanish (es-ES)"""
+    """Spanish"""
```

Comment:
```suggestion
    """Spanish, Spain"""
```

Reply:
Name is directly from the Discord client, so I'm not sure if this should be changed.

Reply:
The standard locale for those codes is the name I wanted to correct it to. But ig Discord just special like that, so nvm :)